web/controllers/auth_controller.ex

```diff
@@ -1,17 +1,26 @@
 require IEx
 defmodule Janitor.AuthController do
   use Janitor.Web, :controller
-  alias Janitor.User 
+  use Timex
+  alias Janitor.User
 
   def connect(conn, _params) do
     redirect conn, external: Google.authorize_url!(scope: "email profile")
   end
 
-  def oauth(conn, %{"code" => code}) do 
+  def oauth(conn, %{"code" => code}) do
     params = token(code) |> get_user! |> map_params
     changeset = User.registration_changeset(%User{}, params)
-    User.find_or_create(changeset)    
-  end 
+    token = User.find_or_create(changeset) |> sign_jwt_token
+    conn = put_req_header(conn, "authorization", "bearer #{token}")
```

Comment:
@mikolajgrzyb It's a temp solution, I didn't know how exactly do you want the token sent to the frontend app.